### DISCUSSION TITLE
討厭和喜歡的「表情符號」

### DISCUSSION THREAD

**Discussion by LinChloe** [1101-GS4719-C/Python-Community-GS4719-C#1958] [General]
為了更好表達我的心情，我會在回訊息時後面加上表情符號。
在此公布我心裡的排名(個人意見，不包含任何立場)

# 喜歡

![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/124855741/ada2ee76-db53-4770-a4b5-848afdb52896)
這個綠綠充滿膠原蛋白的臉頰，真的非常的可愛。
還有可愛的腮紅，跟那用全臉表達噁心的態度令我非常喜歡。

![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/124855741/02d15a4c-cae8-4655-b288-8fbe2b14ecc8)
這安詳的表情，有種認可的feel，感覺是個很有智慧的阿公(大賢者)在同意我的看法。

# 討厭

![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/124855741/e584158a-1c90-4958-ad18-60dd510ef1f5)
通常發這個都是做錯事的時候，所以我看到這個都想一拳揍下去。

![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/124855741/00d71ef6-5dff-47f5-a7c2-4b64c658535b)
打從心裡討厭這玩意，要哭不哭的，滿滿的情勒感湧上來到我都快反胃了。
傳這個的人是朋友的我會想已讀，陌生人的話再也不見。

歡迎大家表達自己的看法，
告訴我你喜歡/討厭的表情貼吧 !!!

**Comment by 110101501** [comment]
我也很喜歡用😌這個表情！！！有一種肯定對方或是猜到對法的想法的得意樣貌。
除此之外也喜歡用😵‍💫或😮‍💨，不論是表達正向意涵或是負向意涵都可以適用。
雖然我沒有特別討厭的符號，但我想分享就是因為自己iOS的版本過低，導致有些符號像是融化、愛心手勢等，根本就看不到😮‍💨

**Comment by wanka2030** [comment]
😶‍🌫️這表情符號對我來很適合用在許多地方，像是不想要跟其他人溝通的時候，將自己隱藏起來並好好放鬆便可以沉浸於怡然之中。
🤬而這是我最常用來表達自己的憤怒，供參。

**Comment by thomasxshen** [comment]
表情符號提供了一種快速而直觀的方式來表達情感和態度。許多人喜歡使用微笑😊或大笑😂表情符號來傳遞正面情緒，而心形❤️符號則用來表達愛和喜悅。相反地，有些人可能不喜歡使用生氣😠或哭臉😢符號，因為它們傳達了負面的情感。然而，每個人對表情符號的偏好都是獨特的，並且可能會隨著情境和個人經歷而變化。重要的是要記住，表情符號應該被用來增強溝通，而不是替代言語。

**Comment by 20050201Shawn** [comment]
我個人蠻少用表情貼的，要用大部分是用👉🏿👈🏿和🤡，蠻討厭😂🤣😅😊🤭，看起來蠻像在嘲諷或反串的。

**Comment by yxin214** [comment]
你的喜歡和討厭和我完全相反...個人超愛用你討厭的那兩個表情符號，第一個覺得他有在傳達很拍謝的意思，而第二個我通常用在很感謝別人的時候。至於你喜歡的第二個，我覺得他帶點笑裡藏刀的感覺。

**Comment by YuanWang08** [comment]
我最喜端的表情符號是😊，不管在什麼情境下都可以用，而且可以給別人一種毛骨悚然的感覺，當對方不知道在說什麼時，就可以給他一個不失禮貌的微笑，讓他自己去想。

**Comment by yyujimmy** [comment]
😊 - 快樂
😔 - 悲傷
😄 - 開心
😅 - 尷尬
😂 - 笑哭
😡 - 生氣
😴 - 睡覺
😒 - 不滿
😕 - 困惑
😎 - 酷

**Comment by KH1126** [comment]
我最喜歡用😂😎😥😡來表達我的情緒，我認為這四個我表情包蠻好用的。

**Comment by jim1743** [comment]
我在與朋友聊天的時候，最常使用的是😮這個驚訝的表情符號，對於不知道要怎麼回復朋友傳過來的訊息時，這個既表達我很驚訝，又能理解他想表達的事，對我來說很方便。我也很討厭那個又哭又微笑的貼圖，不知道為什麼很想揍這個臉。

**Comment by cassy0201** [comment]
只要符合當下情境的都喜歡哈哈哈

**Comment by 710388** [comment]
最討厭🙃，看到覺得非常反感

**Comment by yubo0oo** [comment]
我最討厭笑哭的表情符號
還有咖波
我認識一個怪咖她最喜歡用這兩個
